source/sass/print.scss

```diff
@@ -0,0 +1,58 @@
+@media print {
+  @page {
+    margin: 2cm;
```

Comment:
do we want to make this a percentage instead? Because A4, Letter, and Legal all have different physical dimensions

Reply:
@sabrinang we can predefine page margin for print